server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const next = require('next');
+const dotenv = require('dotenv');
+
+dotenv.config(); // Load environment variables from .env file
+
+const dev = process.env.NODE_ENV !== 'production';
```

Comment:
please use exported env variables instead of directory accessing procce.env

Reply:
removed